tests/scripts/tests-setup-multihost.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -x
+
+function setup_worker(){
+    if [ "$SETUP_SSH" == "yes" ]; then
+        dnf install openssh-server passwd -y
+        
+        # root access is needed in order to access BlueChi's API
+        echo "PasswordAuthentication yes" >> /etc/ssh/sshd_config
+        echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
+
+        systemctl restart sshd
+
+        if [ -z "${SSH_USER}" ]; then
+            SSH_USER="root"
+        fi
+        if [ -z "${SSH_PASSWORD}" ]; then
+            SSH_PASSWORD="root"
+        fi
+        echo "$SSH_PASSWORD" | passwd "$SSH_USER" --stdin
+    fi
+}
+
+function setup_executor(){
```

Comment:
Could you please extract installed dependencies into a standalone script, so it can be reused in both: single and multi host setup?

Reply:
I refactored the original `tests-setup.sh` to provide appropriate functions.